contexts/app-context.tsx

```
@@ -134,35 +136,23 @@
+        /*
+        TODO: Replace with real data
         const serverChannels = mockChannels.filter((channel) => channel.serverId === activeServer)
         setChannels(serverChannels)
 
         if (serverChannels.length > 0 && (!activeChannel || !serverChannels.find((c) => c.id === activeChannel))) {
           setActiveChannel(serverChannels[0].id)
         }
+          */
       } catch (error) {
         console.error("Error fetching channels:", error)
       }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)